bot/seasons/evergreen/recommend_game.py

```diff
@@ -0,0 +1,52 @@
+import json
+import logging
+import os
+from pathlib import Path
+from random import shuffle
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+DIR = "bot/resources/evergreen/game_recs"
+game_recs = []
+
+
+class RecommendGame(commands.Cog):
+    """Commands related to recommending games."""
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.populate_recs()
+
+    @commands.command(name="recommend_game")
+    async def recommend_game(self, ctx):
+        """Sends an Embed of a random game recommendation."""
+        if not game_recs:
+            self.populate_recs()
+            game = game_recs.pop()
+        else:
+            game = game_recs.pop()
+        author = self.bot.get_user(int(game['author']))
+
+        # Creating and formatting Embed
+        embed = discord.Embed(color=discord.Colour.blue())
+        embed.set_author(name=author.name, icon_url=author.avatar_url)
+        embed.set_image(url=game['image'])
+        embed.add_field(name='Recommendation: ' + game['title'] + '\n' + game['wikiLink'], value=game['recText'])
+
+        await ctx.send(embed=embed)
+
+    def populate_recs(self):
```

Comment:
Going along with my other suggestion, the files will only need to be read once so this code could be moved to the top of the module.